src/initialization/InitElement.cpp

```diff
@@ -455,6 +458,27 @@ namespace detail
                     read_element(sub_element_name, m_lattice, nslice_default, mapsteps_default);
                 }
             }
+        } else if (element_type == "linear_map")
+        {
+            auto a = detail::query_alignment(pp_element);
+
+            elements::LinearTransport::Map6x6 transport_map = elements::LinearTransport::Map6x6::Identity();
+
+            // safe to ParmParse inputs for reproducibility
+            for (int i=1; i<=6; ++i) {
+                for (int j=1; j<=6; ++j) {
+                    std::string name = "R" + std::to_string(i) + std::to_string(j);
+                    pp_element.queryAddWithParser<amrex::ParticleReal>(name.c_str(), transport_map(i, j));
+                }
+            }
+            ablastr::warn_manager::WMRecordWarning(
+                "ImpactX::read_element",
+                "Caution, a user-provided linear map is used in '" +
+                element_name + "'. Transport may not be symplectic.",
+                ablastr::warn_manager::WarnPriority::low
+            );
```

Comment:
@cemitch99 should we really raise this at runtime (verbose) or is this something we rather document in the element in the manual?

If we want to raise it at runtime, we need to add it also for Python, probably by moving it from here to the constructor.

My feeling after running the tests is that this is a big verbose and maybe belongs rather in `python.rst` and `parameters.rst` in the docs?

Reply:
I agree that this is something we should document in the element in the manual, and I don't think these runtime warnings are needed.